source/blink.cpp

```diff
@@ -284,8 +286,17 @@ void blink::application::run()
 					print("Finished compiling \"" + object_file.string() + "\" with code " + exit_code + ".");
 
 					// Only load the compiled module if compilation was successful
-					if (exit_code == "0")
-						link(object_file);
+					if (exit_code == "0") {
+						// Whatever happens while linking, we do not want to have the *.obj file in repository.
+						try {
+							link(object_file);
+						}
```

Comment:
Blink doesn't throw exceptions, so this is not necessary. Also, to avoid internal string conversions, it is faster to do `DeleteFileW(object_file.c_str())` =).

Reply:
Forgot about `DeleteFileW`, haha :)